packages/@react-spectrum/icon/docs/custom-icons.mdx

```diff
@@ -0,0 +1,105 @@
+{/* Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/icon';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/icon/package.json';
+
+```jsx import
+import {Flex} from '@react-spectrum/layout';
+import {Icon} from '@react-spectrum/icon';
+```
+
+---
+category: Icons
+---
+
+# Custom Icons
+
+Custom icons can be used by wrapping them in an Icon component.
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Icon']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/iconography/'}
+  ]} />
+
+## Example
+
+```tsx example export=true
+function CustomIcon(props) {
+  return(
+    <Icon {...props}>
+      <svg viewBox="0 0 36 36">
+        <path d="M18.477.593,22.8,12.029l12.212.578a.51.51,0,0,1,.3.908l-9.54,7.646,3.224,11.793a.51.51,0,0,1-.772.561L18,26.805,7.78,33.515a.51.51,0,0,1-.772-.561l3.224-11.793L.692,13.515a.51.51,0,0,1,.3-.908L13.2,12.029,17.523.593A.51.51,0,0,1,18.477.593Z" />
+      </svg>
+    </Icon>
+  );
+}
+
+<CustomIcon aria-label="Star" />
```

Comment:
Should the aria-label be part of the custom icon? `<Icon aria-label="Star" {...props}>` If we did that we'd have to specify an `aria-hidden="true"` in the button example.

Reply:
I think the user could do that if they wanted to, but this seems more useful in case they were to re-use the icon multiple places with different labels. Also the labelling section mentions when you need to add a label. This also matches the `@spectrum-icons/workflow` icons.